src/main/java/mat/client/shared/CQLLibraryResultTable.java

```diff
@@ -47,12 +66,15 @@ public SafeHtml getValue(CQLLibraryDataSetObject object) {
 				return getCQLLibraryNameColumnToolTip(object);
 			}
 		};
-		cqlLibraryName.setFieldUpdater(new FieldUpdater<CQLLibraryDataSetObject, SafeHtml>() {
+//		Double Click event on Grid row to navigate to Composer Tab.
+		table.addDomHandler(new DoubleClickHandler() {
```

Comment:
I think we need to wrap the double click handler in the feature flag as well

Reply:
you are right, I will update this asap.